src/makeInfinityAdder.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const makeAdder = () => {
+    let result = 0;
+
+    const adder = function(number) {
+      if (number == null) {
+        const log = result;
+
+        result = 0;
+
+        return log;
+      } else {
+        result += number;
+
+        return adder;
+      }
+    };
```

Comment:
It's extra, bc you will have false in 13 lines - the code will continue
```suggestion
      }
      
      result += number;

      return adder;
    };
```